models/Asset/Image/Thumbnail/Config/Listing.php

```diff
@@ -16,12 +16,20 @@
 namespace Pimcore\Model\Asset\Image\Thumbnail\Config;
 
 use Pimcore\Model;
+use Pimcore\Model\AbstractModel;
+use Pimcore\Model\Listing\FilterListingInterface;
+use Pimcore\Model\Listing\OrderListingInterface;
+use Pimcore\Model\Listing\Traits\FilterListingTrait;
+use Pimcore\Model\Listing\Traits\OrderListingTrait;
 
 /**
  * @method \Pimcore\Model\Asset\Image\Thumbnail\Config\Listing\Dao getDao()
  */
-class Listing extends Model\Listing\JsonListing
+class Listing extends AbstractModel implements FilterListingInterface, OrderListingInterface
```

Comment:
I would still extend from `JsonListing` on 10.5 since this can lead to BC break, if there's an existing code:
```
$listing = new Asset\Image\Thumbnail\Config\Listing();
if ($listing instanceof JsonListing) {
  $listing ->setFilter(function () { ...
}
```